### DISCUSSION TITLE
Add way to visualize invisible cursor during "No Scope" replays

### DISCUSSION THREAD

**Discussion by OrangeC7** [ppy/osu#16140] [Ideas]
This is a small quality-of-life feature, but I was thinking it would be useful to have some way of visualizing the cursor while watching a no scope replay. Probably the best way to implement this would be to simply keep the cursor from becoming completely transparent during a replay, capping it at something like 85% transparency.

**Comment by bdach** [comment]
Doing *something* special depending on whether there is a replay present is simple enough (check `HasReplayLoaded` in the mod) is simple enough. Primary question is what should happen to the cursor in that case.

**Comment by Joehuu** [comment]
I think this can go in scope with https://github.com/ppy/osu/issues/8357.
